### PR TITLE
Get rid of uiSref and use href instead

### PR DESCRIPTION
**Pull request by uppal101**

### CODE REVIEW

File: packages/api-explorer/src/lib/markdown/renderer.js [hunk 1]
```diff
@@ -78,13 +77,12 @@ renderer.link = function link(href, title, text) {
 
   const blog = href.match(/^blog:([-_a-zA-Z0-9#]*)$/);
   if (blog) {
-    uiSref = `blog.show({'blog': '${blog[1]}'})`;
-    href = '';
+    href = `/blog/${blog[1]}`;
```
Comment (uppal101) [inline packages/api-explorer/src/lib/markdown/renderer.js]: I know you put in the issue to change it to '/blog/:blog', but how will it have access to the variable blog shouldn't we being use string interpolation?

Comment (domharrington) [reply packages/api-explorer/src/lib/markdown/renderer.js]: This line of code looks correct to me?

File: packages/api-explorer/__tests__/lib/markdown/renderer.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      '<a href="/v1.0/blog/getting-started" target="_self">Getting started with petstore swagger</a>',
+    );
+  });
+
+  it('should return a tag with href', () => {
+    expect(renderer.link('/v1.0/page/getting-started', '', '')).toBe(
+      '<a href="/v1.0/page/getting-started" target="_self"></a>',
+    );
+  });
+});
```
Comment (domharrington) [inline packages/api-explorer/__tests__/lib/markdown/renderer.test.js]: You've not added any tests that have `doc:doc` as the href, to test that we're correctly converting those into /doc links.

Can you add one for each type: doc, blog and custompage?

File: packages/api-explorer/src/lib/markdown/renderer.js [hunk 3]
```diff
@@ -53,16 +53,15 @@ renderer.link = function link(href, title, text) {
   /* eslint no-param-reassign: 0 */
   const doc = href.match(/^doc:([-_a-zA-Z0-9#]*)$/);
   let isDoc = false;
-  let uiSref = false;
+  const uiSref = false;
```
Comment (domharrington) [inline packages/api-explorer/src/lib/markdown/renderer.js]: We no longer need this now?

**Comment by domharrington** [comment]
This fixes #29

**Pull request merged**